packages/scheduler/lib/workers/scheduling/scheduling.worker.ts

```diff
@@ -116,11 +116,16 @@ export class SchedulingChild {
                             createdToStartedTimeoutSecs: schedule.createdToStartedTimeoutSecs,
                             startedToCompletedTimeoutSecs: schedule.startedToCompletedTimeoutSecs,
                             heartbeatTimeoutSecs: schedule.heartbeatTimeoutSecs
-                        });
-                        if (task.isErr()) {
-                            logger.error(`Failed to create task for schedule: ${schedule.id}`);
+                        })
+                    );
+                    const res = await Promise.allSettled(createTasks);
```

Comment:
Wait my comment was not sent 🤔
I was saying (before and after this pr) you are not killing the transaction if any tasks fails, is it on purpose?

Reply:
yes it is on purpose. Failing to create a task should not prevent other tasks to be created. 
The same scheduling logic will then create the task in the next iteration if needed